src/styles/components/home-page.scss

```diff
@@ -1,134 +1,101 @@
 .homepage{
-  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Ubuntu,sans-serif;
-  flex: 1;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
   background-image: url('/images/homepage.jpg');
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
-  
+  flex: 1;
 
-  &__wrapper{
+  &__container{
+    height: 100%;
     display: flex;
     flex-direction: column;
     align-items: center;
     backdrop-filter: blur(8px);
     justify-content: center;
-    flex: 1;
-    
   }
 
   
-  &__heading{
+  &__header{
     text-align: center;
   }
-}
 
-.search-wrapper{
-  width: calc(100vw - 50%);
-  height: 200px;
-  background-color: whitesmoke;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  box-shadow: rgb(60 66 87 / 12%) 0px 7px 14px 0px, rgb(0 0 0 / 12%) 0px 3px 6px 0px;
-  border-radius: 8px;
+  &__search-wrapper{
```

Comment:
nice
![Screenshot 2023-05-08 at 21 10 05](https://user-images.githubusercontent.com/2543423/236912300-305322b2-d7c6-4681-80a5-4a8103050de6.png)

probably would be good to make the heights equal
![Screenshot 2023-05-08 at 21 09 35](https://user-images.githubusercontent.com/2543423/236912441-938ec307-cda7-4625-942c-41d84fe523e2.png)